core/kazoo_amqp/src/api/kapi_callflow.erl

```diff
@@ -64,5 +64,10 @@ declare_exchanges() ->
 %%------------------------------------------------------------------------------
 -spec publish_resume(kz_term:api_terms()) -> 'ok'.
 publish_resume(JObj) ->
-    {'ok', Payload} = kz_api:prepare_api_payload(JObj, ?RESUME_VALUES, fun resume/1),
+    {'ok', Payload} = kz_api:prepare_api_payload(JObj
+                                                ,?RESUME_VALUES
+                                                ,[{'formatter', fun resume/1}
+                                                 ,{'remove_recursive', 'false'}
```

Comment:
@jamesaimonetti this went into master but, isn't the below change to `should_strip_from_payload` make this change unnecessary ?

Reply:
Yes and no. We don't really need to recurse into the Flow object so this saves us time, especially if its a big callflow doc.